update/update_2.3.5-2.4.php

```diff
@@ -124,10 +123,19 @@
 	// Add folders at the end of the files list to keep the order (files, folders)
 	$update['items'] = array_merge($update['items'], $folders);
 }
- 
+
 // check version:
-if(!in_array($settings['version'], $update['version'])) {
-	$update['errors'][] = 'Error in line '.__LINE__.': This update file doesn\'t work with the current version.';
+if(!file_exists('../config/VERSION')) {
+	$update['errors'][] = 'Error in line '.__LINE__.': Missing the file config/VERSION.';
+}
+if (empty($update['errors'])) {
+	$newVersion = trim(file_get_contents('../config/VERSION'));
```

Comment:
Wrong path because the update file will be included. The correct path reads `config/VERSION`